packages/apps/purchase/src/i18n/fr.json

```diff
@@ -1,3 +1,13 @@
 {
-  "Purchase_Purchase": "Achats"
+  "Purchase_Purchase": "Achats",
+  "Purchase_InternalRequests": "Demandes internes",
+  "Purchase_Status_Draft": "Brouillon",
+  "Purchase_Status_Requested": "Demandé",
+  "Purchase_Status_Accepted": "Accepté",
+  "Purchase_Status_Purchased": "Acheté",
+  "Purchase_Status_Refused": "Refusé",
+  "Purchase_Status_Canceled": "Annulé",
```

Comment:
I think we should be an ending 'e' on this statuses as "Demande" is a feminine name